plugin/manager_linux.go

```diff
@@ -70,11 +70,12 @@ func (pm *Manager) enable(p *v2.Plugin, c *controller, force bool) error {
 }
 
 func (pm *Manager) pluginPostStart(p *v2.Plugin, c *controller) error {
+	logrus.WithField("plugin", p.Name()).Debug("plugin post start")
```

Comment:
Remove this log. I'm assuming you added it for dev purposes. Dont see any value exposing this in the field.